src/components/Notifications/NotificationSendHistory.tsx

```diff
@@ -23,7 +24,12 @@ const notificationSendHistoryColumns: MRT_ColumnDef<NotificationSendHistoryApiRe
             <Age from={dateString} />
             {(count || 1) > 1 && (
               <span className="inline-block pl-1 text-gray-500">
-                (x{count} over <Age from={firstObserved} />)
+                (x{count} over{" "}
+                <Age
+                  dateTimePreferenceOverride={DateTimePreferenceOptions.Short}
```

Comment:
this should probably be format="short", with the default being format="preference" and the preference coming from a jotai atom lookup inside the Age component. also no need for the constant if the format parameters is typescript `format?: "short" | "long" | "preference" etc..

Reply:
> this should probably be format="short", with the default being format="preference" and the preference coming from a jotai atom lookup inside the Age component.

Isn't it already that way?